src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
 // export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerStatus = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      { age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {(isMarried) ? `${partnerName} is my ${partnerStatus}`
+          : `I am not married`
+        }
+      </p>
```

Comment:
1. Let's add empty line between sibling elements for better readability
```
      <h2 className="Person__name">
         {`My name is ${name}`}
       </h2>

       { age && (
         <p className="Person__age">
           {`I am ${age}`}
         </p>
       )}

       <p className="Person__partner">
         {(isMarried) ? `${partnerName} is my ${partnerStatus}`
           : `I am not married`
         }
       </p>
```

2. Let's format it correctly
```
{
  isMarried
    ? `${partnerName} is my ${partnerStatus}`
    : `I am not married`
}